log/log.go

```diff
@@ -22,26 +21,60 @@ func init() {
 	// Always initializing the logger is also useful to avoid panics when
 	// logging if the logger is nil.
 	level := "error"
-	if s := os.Getenv("LOG_LEVEL"); s != "" {
-		level = s
+	if env, found := os.LookupEnv("LOG_LEVEL"); found {
+		level = env
 	}
+
 	Init(level, "stderr")
 }
 
-func Logger() *zap.SugaredLogger { return log }
-
-// Init initializes the logger. Output can be either "stdout/stderr/filePath"
-func Init(logLevel string, output string) {
-	cfg := newConfig(logLevel, output)
+// Init initializes the logger.
+//
+// logLevels is a comma-separated string like "somename:debug,*:info"
+// (for backwards-compatibility, the "*:" is optional, so "warn" is interpreted as "*:warn")
+//
+// output can be either "stdout", "stderr" or "/path/to/file" and it's a global setting.
+//
+// It's safe to use concurrently.
+func Init(logLevels string, output string) {
```

Comment:
why not just `logLevels []string`? some of the callers use Join, and you use Split here.

Reply:
also, for the sake of clarity, I would call this func `GlobalInit`